templates.json

```diff
@@ -143,6 +150,53 @@
                 "successDebugMessage": "Hello Torizon!"
             }
         },
+        {
+            "img": "unogtk.png",
+            "folder": "dotnetUno",
+            "description": ".NET 6 C# Uno Platform Skia.GTK",
+            "runtime": ".net 6.0",
+            "language": "C#",
+            "debugSSHPort": "2225",
+            "multiContainer": true,
+            "support": "Toradex",
+            "status": "deprecated",
+            "customMessage": "Deprecated due to the EOL of .NET 6.0",
+            "tests": {
+                "debuggers": {
+                    "local": ["Local Debug"],
+                    "amd64": ["Torizon AMD64"],
+                    "arm": ["Torizon ARMv7"],
+                    "arm64": ["Torizon ARMv8"]
+                },
+                "debuggerConsole": null,
+                "mainFile": "__change__.Shared/App.xaml.cs",
+                "mainFileBreakpointLine": 113,
+                "successDebugMessage": "Hello Torizon!"
+            }
+        },
+        {
+            "img": "unofbdrm.png",
```

Comment:
I can see this JSON file becoming a very large file ...

Reply:
Makes sense. Moved to a [new DeprecatedTemplates list](https://github.com/torizon/vscode-torizon-templates/pull/288/files#diff-4559d9c33eeb9a10ed9c3970cab31e7514e354827585f016cfe737fe190379ffR545). Also, removed the `tests` info to make it smaller.